AzureCalling/app/src/main/java/com/azure/samples/communication/calling/helpers/AudioSelectionAdapter.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.samples.communication.calling.helpers;
+
+import android.content.Context;
+import android.widget.PopupWindow;
+import androidx.core.content.ContextCompat;
+import com.azure.samples.communication.calling.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AudioSelectionAdapter extends BottomCellAdapter {
+
+    public AudioSelectionAdapter(final PopupWindow popupWindow,
+                                 final Context context,
+                                 final AudioSessionManager audioSessionManager) {
+        super(context);
+        final AudioDeviceType[] audioDeviceTypes = audioSessionManager.getAudioDeviceTypes();
+        final AudioDeviceType currentAudioDeviceType = audioSessionManager.getCurrentAudioDeviceType();
+        final List<BottomCellItem> bottomCellItems = new ArrayList<>();
+        for (AudioDeviceType audioDeviceType : audioDeviceTypes) {
+            final BottomCellItem bottomCellItem = new BottomCellItem();
+            if (audioDeviceType == AudioDeviceType.ANDROID) {
+                bottomCellItem.setAvatar(ContextCompat.getDrawable(context,
+                        R.drawable.ic_fluent_speaker_2_24_regular));
+                bottomCellItem.setTitle("Android");
+                bottomCellItem.setEnabled(currentAudioDeviceType == audioDeviceType);
+                bottomCellItem.setOnClickAction(() -> setDeviceType(AudioDeviceType.ANDROID,
+                        audioSessionManager, popupWindow));
```

Comment:
I think we should keep the bottomCellItem.set attribute functions since we'll be needing them in the future, but for this particular case, I think we should be using the constructor.  Setting them up individually takes up more cycles, and might make a developer forget or miss an attribute, leading to future bugs.

Reply:
Makes sense and addressed now.